api/report_server.thrift

```diff
@@ -298,6 +305,30 @@ service codeCheckerDBAccess {
                                              4: ReportFilterList reportFilters)
                                              throws (1: shared.RequestFailed requestError),
 
+  // The Counter value should be converted beck to the
+  // proper Severity type by the client.
+  Counters getSeverityCounts(1: list<i64> runIds,
```

Comment:
I am in favour of `CounterInt`. If documented properly, I can live with not having the `enum` as key for the `counter`.

(Wish Thrift would support templates? :innocent: )